### PR TITLE
[10.0][ADD] barcodes_generator_package

### PR DESCRIPTION
**Pull request by lasley**
* Create new module to generate barcodes for product packaging.

Depends:
- [x] #65

### CODE REVIEW

File: barcodes_generator_package/README.rst [hunk 1]
```diff
@@ -0,0 +1,100 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+Generate Barcodes for Packaging
+===============================
+
+This module expends Odoo functionality, allowing user to generate barcode
```
Comment (pedrobaeza) [inline barcodes_generator_package/README.rst]: s/expends/expands

File: barcodes_generator_package/README.rst [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+depending on a given barcode rule for Product Packaging.
+
+For exemple, a typical pattern for partners is  "042........." that means
```
Comment (pedrobaeza) [inline barcodes_generator_package/README.rst]: s/exemple/example

File: barcodes_generator_package/README.rst [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+* a 13 digit control
+
+With this module, it is possible to:
+
+* Affect a pattern (barcode.rule) to a product.packaging
+
+* Define a Barcode base: 
+    * manually, if the base of the barcode must be set by a user. (typically an
+      internal code defined in your company)
+    * automaticaly by a sequence, if you want to let Odoo to increment a
```
Comment (pedrobaeza) [inline barcodes_generator_package/README.rst]: s/automaticaly/automatically

File: barcodes_generator_package/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    "summary": "Generate Barcodes for Product Packaging",
+    "version": "10.0.1.0.0",
+    "category": "Tools",
+    "author": "LasLabs, "
+              "Odoo Community Association (OCA)",
+    "website": "https://www.odoo-community.org",
+    "license": "AGPL-3",
+    "depends": [
+        "barcodes_generator_abstract",
+        "product",
```
Comment (pedrobaeza) [inline barcodes_generator_package/__manifest__.py]: You don't need this explicit dependency

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
You should use an icon with a light variation over the use one, as too many modules will have the same icon. See for example https://github.com/OCA/vertical-association/blob/8.0/membership_prorrate/static/description/icon.png, https://github.com/OCA/vertical-association/blob/8.0/membership_prorrate_variable_period/static/description/icon.png and https://github.com/OCA/vertical-association/blob/8.0/membership_variable_period/static/description/icon.png for the variations I made for three modules that are related.

**Comment by legalsylvain** [comment]
@pedrobaeza : seems that the new commits fixes your remarks.

**Review by legalsylvain — APPROVED** [review]
:+1:

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**